python/graphstorm/dataloading/dataloading.py

```diff
@@ -199,154 +205,167 @@ def __init__(self, dataset, target_idx, fanout,
         """
         return self._data
 
     @property
     def target_eidx(self):
-        """ Target edge idx for prediction
+        """ Target edge indexes for prediction, which is given in class initialization.
 
         Returns
         -------
-        dict of Tensors : the target edge IDs.
+        dict of Tensors: the target edge IDs, which is given in class initialization.
         """
         return self._target_eidx
 
     @property
     def fanout(self):
-        """ The fan out of each GNN layers
+        """ The fan out of each GNN layers, which is given in class initialization.
 
         Returns
         -------
-        list or a dict of list : the fanouts for each GNN layer.
+        list or a dict of list: the fanouts for each GNN layer, which is given in class
+        initialization.
         """
         return self._fanout
 
     @property
     def label_field(self):
-        """ The label field
+        """ The label field, which is given in class initialization.
 
         Returns
         -------
-        str: Label fields in the graph.
+        str: Label fields in the graph, which is given in class initialization.
         """
         return self._label_field
 
     @property
     def node_feat_fields(self):
-        """ Node features
+        """ Node feature fields, which is given in class initialization.
 
         Returns
         -------
-        str or dict of list of str: Node feature fields in the graph.
+        str or dict of list of str: Node feature fields in the graph, which is given in class
+        initialization.
         """
         return self._node_feats
 
     @property
     def edge_feat_fields(self):
-        """ Edge features
+        """ Edge feature fields, which is given in class initialization.
 
         Returns
         -------
-        str or dict of list of str: Node feature fields in the graph.
+        str or dict of list of str: Node feature fields in the graph, which is given in class
+        initialization.
         """
         return self._edge_feats
 
     @property
     def decoder_edge_feat_fields(self):
-        """ Edge features for edge decoder.
+        """ Edge features for edge decoder, which is given in class initialization.
 
         Returns
         -------
-        str or dict of list of str: Node feature fields in the graph.
+        str or dict of list of str: Node feature fields in the graph, which is given in class
+        initialization.
         """
         return self._decoder_edge_feats
 
 class GSgnnEdgeDataLoader(GSgnnEdgeDataLoaderBase):
-    """ The minibatch dataloader for edge prediction
+    """ The mini-batch dataloader for edge prediction tasks.
 
-    GSgnnEdgeDataLoader samples GraphStorm edge dataset into an iterable over mini-batches
-    of samples. Both source and destination nodes are included in the batch_graph, which
+    ``GSgnnEdgeDataLoader`` samples GraphStorm data edges into an iterable over mini-batches
```

Comment:
What does data edges mean?

Reply:
Changed to `target edges` to avoid confusion.